.github/workflows/ubuntu.yml

```diff
@@ -127,30 +129,30 @@ jobs:
           ln -f newcob.val.Z.cached newcob.val.Z
           make EXEC85 && make --jobs=$(($(nproc)+1)) test
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: NIST85 results
+          name: NIST85 results on ${{ matrix.os }}
           path: |
             _build/tests/cobol85/**/*.log
             _build/tests/cobol85/**/*.out
 
       - name: install
         run: |
-          cd _build
-          make install
+          make -C _build install
```

Comment:
This is only reasonable (because it is already done internally in `make distcheck?`) if you want to create a binary package and make it available as artifact, but then `make distbin` may be preferable; if you don't want to provide that artifact (now) then drop that last step in this CI.